R/spectraTab.R

```diff
@@ -1,42 +1,39 @@
 #' tuneParams tab showing size spectra
-#' 
+#'
 #' This tab shows:
-#' 
+#'
 #' * a plot of the biomass density as a function of log size,
-#' including all species and the resource. 
+#' including all species and the resource.
 #' * A slider for scaling the background (resource and possibly background
 #' species).
 #' * If there are background species, buttons to adjust or remove those
 #' background species.
-#' 
+#'
 #' @inheritParams abundanceControl
 #' @param logs Environment holding the log of steady states.
 #' @param trigger_update Reactive value used for triggering update of
 #'   species parameter sliders.
 #' @param ... Unused
 spectraTab <- function(input, output, session,
                        params, logs, trigger_update, ...) {
-    
+
     ## Plot spectra ####
     output$plotSpectra <- renderPlotly({
         # if (input$binning == "Logarithmic") {
-        power <- 2
+            power <- 2
         # } else {
         #     power <- 1
         # }
-        plot <- plotSpectra(params(), power = power, highlight = input$sp, 
-                            total = TRUE) +
-            theme(text = element_text(size = 12))
```

Comment:
It is of course good if you don't need this line and the next any more. But can you explain why?

Reply:
I missed the `element_text` at 12. Added for all the other plots as well. It seems that for the same value, text appears bigger using `ggpplotly` compared to `ggplot`.